src/zc/Zcmp_footer.adoc

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
do we still need the footer?

Reply:
I don't believe we do, as the only footer in the original that is used applies to the Zcb instructions and all it says that the instructions is included in Zcb, which is redundant as the instruction is in the Zcb instruction formats section.